examples/versioning.ipynb

```diff
@@ -0,0 +1,667 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "52fe1001-166a-4356-b6ba-869e76bdd05d",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "'1'\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(f\"'{version_trait.getStableTag()}'\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6485867c-086a-4e97-98b6-c56a08da31ef",
+   "metadata": {},
+   "source": [
+    "The `stableTag` proptery holds the version tag for the targeted entity that is guaranteed to be free of dynamic behaviour. So in this case, indicates that it is version \"1\".\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a05b7b25-6414-4679-8dfa-d34c0a0d24b5",
+   "metadata": {},
+   "source": [
+    "## Querying a specific version"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "62c0dea8-42ed-47a0-b62a-fffe552ddf05",
+   "metadata": {},
+   "source": [
+    "It was noted earlier that Hosts, generally speaking, should never \"string munge\" an existing entity reference to make up another one as this makes assumptions about how any manager may work, which is almost guaranteed to be different for each manager. \n",
+    "\n",
+    "Entity references shold be considerd \"opaque handles\", and instead use the appropriate API methods to achieve the same end.\n",
+    "\n",
+    "Here we're going to use the `getWithRelationship` method with an unversioned entity reference, to query the reference for a specific version of the entity - with the tag `\"1\"` (obtained above).\n",
+    "\n",
+    "To do this, we use the `EntityVersionsRelationshipSpecification` from the `specifications.lifecycle` module, which defines a well-known relationship trait set for querying different versions of the same entity."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "id": "fa2020ad-f6bb-458d-8ab1-8dd9b93dc1ce",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "['bal:///project_artwork/logos/openassetio?v=1']\n"
+     ]
+    }
+   ],
+   "source": [
+    "from openassetio.access import RelationsAccess\n",
+    "\n",
+    "from openassetio_mediacreation.specifications.lifecycle import EntityVersionsRelationshipSpecification\n",
+    "\n",
+    "# Define the relationship we wish to query, and the specific\n",
+    "# tag we want to look up.\n",
+    "relationship_spec = EntityVersionsRelationshipSpecification.create()\n",
+    "relationship_spec.versionTrait().setSpecifiedTag(\"1\")\n",
+    "\n",
+    "# Note that the relationship query methods don't yet have the same\n",
+    "# convenience overloads that resolve does, and so required the use\n",
+    "# of the more elaborate callback signature.\n",
+    "related_refs = []\n",
+    "manager.getWithRelationship(\n",
```

Comment:
[Fix] This needs updated to be the pager version now, not really a problem with the PR, it changed under you.

Reply:
Updated in the latest version.